client/blocks/post-share/nudges.jsx

```diff
@@ -7,48 +7,71 @@ import React from 'react';
  * Internal dependencies
  */
 import Banner from 'components/banner';
-import { PLAN_BUSINESS } from 'lib/plans/constants';
+import { PLAN_PREMIUM, PLAN_JETPACK_PREMIUM } from 'lib/plans/constants';
 import formatCurrency from 'lib/format-currency';
 
 export const UpgradeToPremiumNudge = props => {
 	const {
-		businessDiscountedRawPrice,
-		businessRawPrice,
+		premiumPrice,
+		jetpackPremiumPrice,
 		translate,
 		userCurrency,
+		isJetpack,
 	} = props;
 
+	let price, featureList, proposedPlan;
+	if ( isJetpack ) {
+		price = jetpackPremiumPrice;
+		featureList = [
+			translate( 'Schedule your social messages in advance.' ),
+			translate( 'Easy monetization options' ),
+			translate( 'VideoPress support' ),
+			translate( 'Daily Malware Scanning' ),
+		];
+		proposedPlan = PLAN_JETPACK_PREMIUM;
+	} else {
+		price = premiumPrice;
+		featureList = [
+			translate( 'Schedule your social messages in advance.' ),
+			translate( 'Remove all advertising from your site.' ),
+			translate( 'Enjoy live chat support.' ),
+			translate( 'Easy monetization options' ),
+			translate( 'Unlimited premium themes.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Unlimited Premium Themes' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/publicize/enable-republicize-for-jp-personal)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).